engine/src/main/java/com/datatorrent/stram/client/AppPackage.java

```diff
@@ -49,6 +49,8 @@
 import net.lingala.zip4j.exception.ZipException;
 import net.lingala.zip4j.model.ZipParameters;
 
+import static com.datatorrent.stram.client.StramClientUtils.jsonFileToAppInfo;
```

Comment:
Please just use StramClientUtils.jsonFileToAppInfo instead of having import static.